.env.sample

```diff
@@ -1,4 +1,6 @@
 POSTGRES_NAME
 POSTGRES_USER
 POSTGRES_PASSWORD
-PYTHONDONTWRITEBYTECODE
\ No newline at end of file
+PYTHONDONTWRITEBYTECODE
+POSTGRES_HOST
```

Comment:
why am i seeing the changes that are already merged ?

Reply:
because this branch was created before merging those changes.
so this feature/api_config branch does not have the environment variable changes on the remote, thats why these changes are being merged here so that all branches stay up to date